setup.cfg

```diff
@@ -1,2 +1,12 @@
 [bdist_wheel]
 universal = true
+
+[mypy]
+python_version = 3.7
```

Comment:
This should be not set here as we need to run against both python 2 and 3, given we support both.

Reply:
can remove this. we'd have to set two different runs in tox then, one for each version

Reply:
yeah, can do that in tox.ini as separate commands one after another